staging/src/k8s.io/kms/internal/plugins/_mock/go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/kms/plugins/mock
 
-go 1.24.0
+go 1.25.0
 
 godebug default=go1.24
```

Comment:
```suggestion
godebug default=go1.25
```

Reply:
We might not want to include tools if we do that, though I also think we might want to stop using a monolithic tools module and some of them we would want to keep in sync (linting?)

but eg we probably don't care if yamllfmt or gotestsum or the publishing-bot are able to upgrade language version at the same time.